ava-labs/core-mobile
# CP-9348 Mobile - force logout after exit out forgot pin flow

## Description

**Ticket: [CP-9348]** 

This PR introduces "Slide to confirm" component along with 2 screens that replace modal screens in two use cases:
1.  pressing Forgot PIN takes user to the confirmation screen
2. pressing Delete Wallet in the settings menu takes the user to the confirmation screen

For 1., screen stack had to be made to show new ForgotPin screen.
For 2. SignOutModal is reused and renamed.

## Screenshots/Videos

https://github.com/user-attachments/assets/519d74ef-c843-4619-9a33-2510f5275e1e


https://github.com/user-attachments/assets/04862996-62b7-413e-9315-158eeca03e31


## Testing
Please try these scenarios for both Seedless and Mnemonic wallets:
- Use `Delete Wallet` button from menu to delete wallet
- Use `Forgot Pin` from pin screen when entering wallet
- Check that `Forgot Pin` button is not present when changing pin from `Security & Privacy` menu

## Checklist

Please check all that apply (if applicable)


[CP-9348]: https://ava-labs.atlassian.net/browse/CP-9348?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
